libc-bottom-half/headers/public/__header_sys_socket.h

```diff
@@ -34,6 +34,9 @@
 extern "C" {
 #endif
 
+int accept(int socket, struct sockaddr *restrict addr, socklen_t *restrict addrlen);
+int accept4(int socket, struct sockaddr *restrict addr, socklen_t *restrict addrlen, int flags);
+
```

Comment:
Why do these need to be declared here, but not other socket APIs?   

It looks like `top-half/musl/include/sys/socket.h` can be modified instead to declare these maybe?

Reply:
Sure I will do so

Reply:
Yes.

Reply:
Made the Change